src/redux/configureStore.js

```diff
@@ -0,0 +1,13 @@
+import { configureStore, combineReducers } from 'react-redux';
```

Comment:
- [ ] Kindly use `createStore` instead of `configureStore` , because configureStore is friendly abstraction over the standard Redux `createStore` and its part of [Redux Toolkit](https://redux-toolkit.js.org/) which should be imported like this `import { configureStore } from '@reduxjs/toolkit'`